packages/snaps-cli/src/builders.ts

```diff
@@ -126,10 +127,17 @@ const builders: SnapsCliBuilders = {
     describe:
       'Whether to use Babel to transpile all source code (including dependencies), local source code only, or nothing.',
     demandOption: false,
-    default: TranspilationModes.all,
+    default: TranspilationModes.localOnly,
```

Comment:
We should default this to `localAndDeps` because it is the "safest" default in that it will prevent more errors during the `eval` step.

```suggestion
    default: TranspilationModes.localAndDeps,
```